bin/pic-configure

```diff
@@ -165,6 +165,14 @@ if [ -f "$extension_param/cmakeFlags" ] ; then
         echo "       Is the file executable? (chmod u+x cmakeFlags)" >&2
         exit 2
     fi
+    # save the cmakeFlags setup number for future reference
+    if [ -f "$extension_param/cmakeFlagsSetup" ] ; then
+      rm $extension_param/cmakeFlagsSetup
+    fi
+    touch $extension_param/cmakeFlagsSetup
+    echo -e "Last configured cmakeFlags setup was the setup No.: $cmakeFlagsNr" > $extension_param/cmakeFlagsSetup
+    grep "^flags\[$cmakeFlagsNr\]" $extension_param/cmakeFlags >> $extension_param/cmakeFlagsSetup
```

Comment:
Please use the line I suggested [here](https://github.com/ComputationalRadiationPhysics/picongpu/pull/3537#discussion_r595154549)

The file `cmakeFlags` is a bash script, there is no need that someone is adding an array you ca find with grep. A generator is also possible or `flags+="new entry"`

The variable `cmake_flags` contains already the content of the selected setup.